ReactWindows/ReactNative.Tests/Modules/Network/NetworkingModuleTests.cs

```diff
@@ -363,6 +365,7 @@ public void NetworkingModule_Response_Content_UseIncremental()
                     }
                     else if (eventName == "didCompleteNetworkResponse")
                     {
+                        onCompleteData = args[1] as JArray;
```

Comment:
It would be better to insert a Assert.IsNotNull on the onCompleteData before having a null exception, yes.  But an exception thrown here can be thrown from a different thread, causing the process to crash rather than the test to fail (because it's a continuation from an async operation)